src/api/views/domain_views.py

```diff
@@ -292,9 +292,15 @@ def post(self, domain_id):
             post_data = request.json
             post_data["domain"] = domain_name
 
+            is_go_template = template.get("is_go_template")
+            if is_go_template:
+                is_template = "true"
+            else:
+                is_template = "false"
+
             # Generate website content from a template
             resp = requests.post(
-                f"{STATIC_GEN_URL}/generate/?category={category}&domain={domain_name}",
+                f"{STATIC_GEN_URL}/generate/?template_name={template_name}&domain={domain_name}&is-template={is_template}",
```

Comment:
should be is_template

Reply:
Actually I'd pass it in as is_go_template and do `is_go_template = template.get("is_go_template", True)`